.github/workflows/mainnet.yaml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      DAI_ACCOUNT_HOLDER_FORKED_NETWORK: '0x4a75f0ae51a5d616ada4db52be140d89302aaf78'
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
```

Comment:
we should probably use `npm ci` instead of `install`.

https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
sure but for indivdual packages like ganache and truffle I think the install keyword will be required right ?

Reply:
yep done you can check the latest commit